README.md

```diff
@@ -35,6 +35,18 @@ and delete this line:
 
 then run brew install giter8 again.
 
+# Provide compressed images
+Every blogpost has at least one image which makes the Lunatech blog relatively heavy in memory consumption.
+One way of compressing images is by using `pngcrush`. You can install it using brew:
```

Comment:
Might be nice to add a follow-up mention that compressing combats this memory consumption.

One way of to combat this is by compressing images via [pngcrush](https://pmt.sourceforge.io/pngcrush/).
Installing it as follows:

Reply:
Good point